packages/rum-core/src/domain/rumEventsCollection/view/viewMetrics/trackFirstInputTimings.ts

```diff
@@ -32,15 +42,25 @@ export function trackFirstInputTimings(
     )
     if (firstInputEntry) {
       const firstInputDelay = elapsed(firstInputEntry.startTime, firstInputEntry.processingStart)
+      let firstInputTargetSelector
 
-      webVitalTelemetryDebug.addWebVitalTelemetryDebug('FID', firstInputEntry.target, firstInputEntry.startTime)
+      if (
+        isExperimentalFeatureEnabled(ExperimentalFeature.WEB_VITALS_ATTRIBUTION) &&
+        firstInputEntry.target &&
+        isElementNode(firstInputEntry.target)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏how about adding a test when the target is not an element?